### PR TITLE
fix(sheet): fix some copy/paste bugs

### LINKED ISSUE #1619: [Bug] When pasting into merged cells, the expected prompt cannot be pasted, and the columns are not expanded after pasting when there are insufficient columns.
### 在您提交此问题之前，您是否检查了以下内容？

- [X] 这真的是个问题吗？
- [X] 我已经在 Github Issues 中搜索过了，但没有找到类似的问题。

### 受影响的包和版本

dev

### 复现链接

1、复制单元格(1、2)，粘贴到区域A
2、复制单元格(1、2)，粘贴到区域B
image

### 预期行为

1、有禁止粘贴提示
2、列扩展，可以粘贴

### 实际行为

[2024_3_16 17_01_19 commands.json](https://github.com/dream-num/univer/files/14623016/2024_3_16.17_01_19.commands.json)
[Uploading 2024_3_16 17_01_19 video.webm…]()

### 运行环境

_No response_

### 系统信息

_No response_

### PR DESCRIPTION
**Pull request by ybzky**
修复
1. 复制时卡顿问题
2. 粘贴飞书表格时携带comment
3. 粘贴excel没有携带边框线、字色、背景颜色、字重；字号pt转px错误
4. 粘贴没有解析数字格式
5. 复制生成的表格行列数不正确
6. set-border type为0仍然渲染问题
7. 增加外部表格粘贴进来没有解析富文本的逻辑
8. 修复从excel粘贴字号大小不正确的问题
9. 粘贴区域的合并单元格没有被清除的问题
10. 粘贴时行高错误

&

close #1619 #1624 #1585 #1559 #1209 #1082 #1389

### CODE REVIEW

**Comment by wzhudev** [comment]
Please build unit tests so we can make sure that these problems wouldn't occur in the future.

File: packages/sheets-ui/src/services/clipboard/html-to-usm/converter.ts [hunk 1]
```diff
@@ -546,3 +439,212 @@ function decodeHTMLEntities(input: string): string {
...
+        })
+        .filter((style) => style)
+        .join(' ');
+
+    newStyleString += ` ${withoutBorders.trim()}`;
+
+    return newStyleString.trim();
+}
+
+function parseTableByHtml(htmlElement: HTMLIFrameElement, skeleton?: SpreadsheetSkeleton) {
```
Comment (yuhongz) [inline packages/sheets-ui/src/services/clipboard/html-to-usm/converter.ts:472]: This method is too long, and it requires additional comments to improve its understanding.

**Review by yuhongz — APPROVED** [review]
LGTM

**Comment by zhaolixin7** [comment]
@ybzky #1619 粘贴列不够时未扩展

**Comment by wzhudev** [comment]
Please rebase this PR.

**Pull request merged**